submissions/adeolaFafemi/dom/productCost.js

```diff
@@ -0,0 +1,58 @@
+// selecting elements
+var productPrice = document.getElementById('productprice');
+var shippingCost= document.getElementById('shippingcost');
+var totalCost = document.getElementById('totalcost');
+var input = document.getElementById('quantity');
+var errorInput = document.getElementById('error-input');
+var buyNow = document.getElementById('submit')
+var alphaExp = /^[0-9]+$/;
```

Comment:
This variable name is confusing. `alpha` sounds like it is alphabetical expression